src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeLayout.cs

```diff
@@ -16,15 +20,36 @@ public class ThemeLayout : Razor.RazorPage<dynamic>
     {
         public override async Task ExecuteAsync()
         {
-            // The View's body is rendered 
+            // The View's body is rendered
             var body = RenderLayoutBody();
 
             if (ThemeLayout != null)
             {
                 // Then is added to the Content zone of the Layout shape
                 ThemeLayout.Content.Add(body);
 
-                // Finally we render the Shape's HTML to the page's output
+                // Render Shapes in Content
+                if(ThemeLayout.Content is Shape content)
```

Comment:
So, because now we don't use the shape.Items anymore, hee we can just use `IShape`

Reply:
change to IShape